cypress-tests/cypress/support/commands.js

```diff
@@ -273,12 +279,15 @@ Cypress.Commands.add("createConfirmPaymentTest", (createConfirmPaymentBody, deta
       "api-key": globalState.get("apiKey"),
     },
     body: createConfirmPaymentBody,
+    failOnStatusCode: false,
   }).then((response) => {
 
     logRequestId(response.headers['x-request-id']);
 
     expect(response.headers["content-type"]).to.include("application/json");
+    if (response.body.hasOwnProperty("status")) {   
     expect(response.body).to.have.property("status");
+    }
```

Comment:
can we have a check for status code, if it is 2xx, they we always expect status, if 4xx, we ll always expect error code and message